config/notifiers.go

```diff
@@ -59,7 +59,7 @@ var (
 		NotifierConfig: NotifierConfig{
 			VSendResolved: false,
 		},
-		Color:     `{{ if eq .Status "firing" }}danger{{ else }}good{{ end }}`,
+		Color:     `{{ template "slack.default.color" }}`,
```

Comment:
For future spelunkers, this is wrong. It should be `{{ template "slack.default.color" .}}`. That last dot is very important.